fe/src/main/java/org/apache/doris/catalog/Column.java

```diff
@@ -310,6 +333,50 @@ public static String removeNamePrefix(String colName) {
         return colName;
     }
 
+    public Expr getDefineExpr() {
+        return defineExpr;
+    }
+
+    public void setDefineExpr(Expr expr) {
+        defineExpr = expr;
+        if (expr != null) {
+            serializeDefineExpr(expr);
+        }
+    }
+
+    public void serializeDefineExpr(Expr defineExpr) {
+        defineExprString = "COLUMNS (" + name + "=" + defineExpr.toSql() + ")";
```

Comment:
Actually, `toSql()` is really not a safe way for serialization. It may be change, or even can not restore the original statement, in case the expr may be much complicate and being rewritten somehow.

I was thinking maybe a safer way is to save the entire `create view` statement directly. We shall discuss this later.

Reply:
You need to restore the original statement rather then use the toSql function